share/jupyter/voila/templates/base/static/main.js

```diff
@@ -52,11 +52,12 @@ require([window.voila_js_url || 'static/voila'], function(voila) {
             // it seems if we attach this to early, it will not be called
             const matches = document.cookie.match('\\b_xsrf=([^;]*)\\b');
             const xsrfToken = (matches && matches[1]) || '';
+            const  configData = JSON.parse(document.getElementById('jupyter-config-data').textContent);
```

Comment:
Minor formatting edit to remove an extra space:

```suggestion
            const configData = JSON.parse(document.getElementById('jupyter-config-data').textContent);
```